ext/pdo_sqlite/sqlite_driver.c

```diff
@@ -224,8 +224,27 @@ static zend_string* sqlite_handle_quoter(pdo_dbh_t *dbh, const zend_string *unqu
 {
 	char *quoted;
 	if (ZSTR_LEN(unquoted) > (INT_MAX - 3) / 2) {
+		php_error_docref(NULL, E_WARNING, "String is too long to be quoted");
```

Comment:
I don't think adding warnings is appropriate?

Reply:
I actually think a warning here is appropriate: when quote() fails, an error of some kind should be generated to explain why quote failed. Having no error generated is inappropriate IMO

Reply:
Right, I overlooked this was targeted to master